data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,268 @@ event "remnant: pantica void sprites"
+				goto dawndetails
+			label dawnsymbiont
+			choice
+				`	"Okay, I will meet you onboard the ship."`
+					accept
+				`	"Developed quite the appetite, haven't you?"`
+			`	She nods. "Yeah, I am not yet used to the nutrient intake required to maintain my symbiont, so I have to consciously make sure that I eat enough to sustain it."`
+			choice
+				`	"Interesting. I'll see you onboard the <ship>."`
+					accept
+				`	"Could you tell me more about them?"`			
+			label dawndetails
+			apply
+				"remnant symbionts info" += 1
+			`	"Basically, ages ago we domesticated some of the creatures found here, and discovered that they could be attached to us to form a symbiotic relationship." She pauses again to eat. "I only recently received mine, so I am not fully adjusted to it yet."`
+			choice
+				`	"Okay, that's as much as I need to know."`
+					goto onwards
+				`	"What is it like to have a symbiont?"`
+			`	"Oh, it is wonderful. It is very hard to describe. At first it is just slimy, then a lot of tingling. The tingling lasted for several hours, and then it was just like waking up: rising up out of darkness to a brighter world with just so much to see and learn. Suddenly, it was as if everything was moving slower and I could take a long hard look at everything..." Dawn is visibly excited, signing so fast you have trouble following.`
+			choice
+				`	"Thanks for the description. I should go prep the <ship> now."`
+					accept
+				`	"Where is your symbiont? Yours does not seem to be obvious."`
+					goto dawnwhere
+				`	"Could I see your symbiont?"`
+			`	Dawn looks at you with an amused expression. "We don't know each other nearly well enough for that. Better to focus on the expedition. I will catch up to you on the ship."`
+				accept
+			label onwards
+			`	She nods. "Well, I will finish my meal and meet you on the <ship> shortly." With that, she returns to her meal.`
+				accept
+			label dawnwhere
+			`	Dawn looks at you with an amused expression. "Just as every human is a unique individual, so every symbiont is also unique. As such, they attach to different areas of the body. Mine is located primarily along my spine." She twists around, and you can see a small layer of the tell-tale scaly leather extending up the back of her neck.`
+			`	"We should probably be focusing on the expedition, though. You go get the <ship> ready for launch, and I will be there shortly."`
+				accept
+	on accept
+		event "remnant: lab availability"
+	on complete
+		outfit "Research Laboratory"
+		conversation
+			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors they pick it up and maneuver it inside. They start securing it to the ship while one approaches you and Dawn.`
```

Comment:
```suggestion
			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors, they pick it up and maneuver it inside. They start securing it to the ship while Dawn and one of the scientists approaches you.`
```
It isn't really established that Dawn is back before it seems like she just suddenly appears next to you. Having Dawn walk up with the scientist resolves the concern.